source/conversions.tex

```diff
@@ -524,21 +524,21 @@
 object. The null pointer value is converted to the null pointer value of
 the destination type.
 
-\rSec1[conv.mem]{Pointer to member conversions}
+\rSec1[conv.mem]{Pointer-to-member conversions}
```

Comment:
We have 15 appearances where "pointer-to-member" is, in fact, hyphenated, so I presume you would accept a patch that removes the hyphens there?